integ/components/deadline/deadline_05_secretsManagement/test/deadline_05_secretsManagement.test.ts

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as CloudFormation from 'aws-sdk/clients/cloudformation';
+import * as SecretsManager from 'aws-sdk/clients/secretsmanager';
+import * as AWS from 'aws-sdk/global';
+import awaitSsmCommand from '../../common/functions/awaitSsmCommand';
+
+// Name of testing stack is derived from env variable to ensure uniqueness
+const testingStackName = 'RFDKInteg-SM-TestingTier' + process.env.INTEG_STACK_TAG?.toString();
+
+const cloudformation = new CloudFormation();
+const secretsManager = new SecretsManager();
+
+const bastionRegex = /bastionId/;
+const smSecretRegex = /deadlineSecretsManagementCredentialsSM(\d)/;
+
+const testCases: Array<Array<any>> = [
```

Comment:
With only one test case it looks like over complicated. Do you expect another test cases here?

Reply:
Good point. I left this in mostly to conform to the format of the other integ tests we have, but since we don't have any other test cases right now this can be removed. Updated.